integration/docker/kill_test.go

```diff
@@ -96,23 +97,29 @@ var _ = Describe("docker kill", func() {
 		},
 		withSignal(syscall.SIGHUP, canBeTrapped),
 		withSignal(syscall.SIGINT, canBeTrapped),
-		withSignal(syscall.SIGQUIT, cannotBeTrapped), //131
-		withSignal(syscall.SIGILL, cannotBeTrapped),  //132
+		// This is not forwarded by cc-shim https://github.com/clearcontainers/runtime/issues/769
+		//withSignal(syscall.SIGQUIT, canBeTrapped),
+		// This is not forwarded by cc-shim https://github.com/clearcontainers/runtime/issues/769
+		// withSignal(syscall.SIGILL, canBeTrapped),
 		withSignal(syscall.SIGTRAP, canBeTrapped),
 		withSignal(syscall.SIGIOT, canBeTrapped),
-		withSignal(syscall.SIGBUS, cannotBeTrapped),  //135
-		withSignal(syscall.SIGFPE, cannotBeTrapped),  //136
+		// This is not forwarded by cc-shim https://github.com/clearcontainers/runtime/issues/769
+		// withSignal(syscall.SIGBUS, canBeTrapped)
```

Comment:
can you skip them? I'd like to see what signals are skipped

Reply:
@devimc  test are skipped.